src/test/java/org/openmrs/contrib/qaframework/RunTest.java

```diff
@@ -40,6 +40,7 @@
 		public static final String REPORT = "@report";
 		public static final String VITALS = "@vitals";
 		public static final String FIND_PATIENT = "@findPatient";
+		public static final String MARK_PATIENT_DECEASED = "@markpatientdeceased";
```

Comment:
The problem is because you are using @selenium tag in your feature file when you haven't defined it to be used under your RunTest class.
Add this line in your run test class   
public static final String **SELENIUM_MARK_PATIENT_DECEASED = SELENIUM + " and " + MARK_PATIENT_DECEASED;**,   Then use  this as your hook **SELENIUM_MARK_PATIENT_DECEASED**